### PR TITLE
Add bug id to changelog of pillar example fix

### PR DESCRIPTION
**Pull request by smpsgill**
Add missing bugzilla reference for fix. Related to https://github.com/SUSE/habootstrap-formula/pull/54

### CODE REVIEW

File: habootstrap-formula.changes [hunk 1]
```diff
@@ -3,6 +3,7 @@ Thu Aug 20 00:56:44 UTC 2020 - Simranpal Singh <simranpal.singh@suse.com>
 
 - Version bump 0.3.8
   * Include the pillar example file in package
+  (bsc#1174994)
```
Comment (MalloZup) [inline habootstrap-formula.changes:6]: I think it is better to put it on same line of description  instead of newline

**Review by arbulu89 — APPROVED** [review]

**Review by MalloZup — APPROVED** [review]
i'm ok too. I used to put bsc number inline but there is not strong convention there

**Review by arbulu89 — CHANGES_REQUESTED** [review]
@Simranpal I will request changes to fix the folder where the `pillar.example` file is stored. I would store it in the `doc` folder

**Comment by smpsgill** [comment]
> @Simranpal I will request changes to fix the folder where the `pillar.example` file is stored. I would store it in the `doc` folder

As discussed in the other PR, placing it in `metadata` directory

**Review by arbulu89 — APPROVED** [review]

**Pull request merged**